content/03.use-cases-a.md

```diff
@@ -10,14 +10,15 @@ A centralized research compendium stored in a version-controlled repository has
 The ability to make a GitHub repository private is particularly advantageous for storing materials without necessarily sacrificing privacy, which may be preferred or even essential for certain compendia. 
 Researchers can then grant specific collaborators read and/or write access to private repositories to pursue pre-publication analyses or writing in privacy, or even to collaborate on research that will permanently remain private.
 
+<!-- Is this paragraph too complicated/jargony?-->
```

Comment:
I think it's fine

Reply:
I think the first sentence is hard to follow, but could easily be remedied by changing the sentence structure a bit e.g. "Additionally, users have the ability to make their repository private, allowing researchers to store materials without sacrificing privacy which may be preferred, or essential for certain subject matter"